src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -12,41 +14,22 @@ public static class ExpectedExceptionTests
     [OuterLoop]
     public static void NotExistentHost_Throws_EndpointNotFoundException()
     {
-        string nonExistHost = "http://nonexisthost/WcfService/WindowsCommunicationFoundation";
-        //On .NET Native retail, exception message is stripped to include only parameter
-        string expectExceptionMsg = nonExistHost;
+        string nonExistentHost = "http://nonexisthost/WcfService/WindowsCommunicationFoundation";
 
-        try
-        {
-            //This test can also hang for other test infrastructure related reasons, adding clock based timeout in addition to the innner SendTimeout.
-            DateTime start = DateTime.Now;
-            while (DateTime.Now.Subtract(start).Seconds < 15)
-            {
-                BasicHttpBinding binding = new BasicHttpBinding();
-                //Setting a timeout as this test takes 500 seconds to finish when it fails.
-                binding.SendTimeout = TimeSpan.FromMilliseconds(10000);
-                using (ChannelFactory<IWcfService> factory = new ChannelFactory<IWcfService>(binding, new EndpointAddress(nonExistHost)))
-                {
-                    IWcfService serviceProxy = factory.CreateChannel();
-                    string response = serviceProxy.Echo("Hello");
-                }
-            }
-        }
-        catch (Exception e)
-        {
-            if (e.GetType() != typeof(EndpointNotFoundException))
-            {
-                Assert.True(false, string.Format("Expected exception: {0}, actual: {1}", "EndpointNotFoundException", e.GetType()));
-            }
+        BasicHttpBinding binding = new BasicHttpBinding();
+        binding.SendTimeout = TimeSpan.FromMilliseconds(10000);
```

Comment:
Ah, I see what you mean now. Wouldn't each test would now have to be aware of this number? 

I feel like this might be something we push for in the xunit harness instead :)